src/styles/main.scss

```diff
@@ -0,0 +1,135 @@
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
```

Comment:
There is a missing unit for the `min-width` value in the media query. It should be `1024px` instead of `1024`.